share/kcr/commands/fzf/kcr-fzf-grep

```diff
@@ -11,10 +11,13 @@
 
 # Not pretty but robust.
 # https://github.com/junegunn/fzf/issues/2581
-export PATHS_PATH=$(mktemp)
+export PATHS_PATH=$(mktemp -d)
```

Comment:
Have you tested the code?  `-d` will create a directory and paths will be written to the directory.

Reply:
apparently I misunderstood you yesterday and this is nonsense
removed commit

Reply:
removed `export`